packages/common/package.json

```diff
@@ -24,7 +24,7 @@
   },
   "scripts": {
     "build-docs": "npm run tsc && ../../node_modules/.bin/jsdoc2md --template templates/docs-API.hbs URLUtils.js util.js > docs/API.md",
-    "clean": "rm -f *.d.ts *.map",
+    "clean": "rm -f *.d.ts *.map && ls *.js | grep -v nyc.config.js | xargs rm -f",
```

Comment:
👍    We're going to need to be mindful of this behavior if adopted as a pattern, as there are <other things?> that use .js configs out there.

Reply:
This should be safer: 380445e91c8afe5c143440b2620e3b747dea16e8